### DISCUSSION TITLE
网页里的文档附件显示乱码

### DISCUSSION THREAD

**Discussion by woboo110** [F9y4ng/GreasyFork-Scripts#380] [问答专区(Question / Answer)] [answered]
网页里的文档附件显示乱码，求教该排除什么元素？
![1729149371153](https://github.com/user-attachments/assets/6045056c-3397-4070-b6ce-7b459decf519)

**Comment by cxw620** [comment, marked as answer]
我记得有 issue 说过, 取消渲染画布看看
![图片](https://github.com/user-attachments/assets/f9f518bc-97da-404e-8971-9bfe9a7473b0)

**Reply by woboo110** [reply]
感谢。解决了
